src/roles/smart_proxy/tasks/main.yaml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
We used to have a `foreman_proxy` role (removed in https://github.com/theforeman/foremanctl/pull/81), it also had tests, which we should bring back :)

Reply:
Reminds me of the whole should we lean into smart-proxy or foreman-proxy as the naming.

Reply:
So would I.